lark/exceptions.py

```diff
@@ -136,10 +137,13 @@ def _format_expected(self, expected):
 
 
 class UnexpectedEOF(ParseError, UnexpectedInput):
```

Comment:
Wasn't lalr not raising this exception? Seems also like a good change for 1.0.

Reply:
I agree. But that's for another PR.